src/Language/ASKEE/Exposure/Interpreter.hs

```diff
@@ -124,10 +130,13 @@ suspend v
 data EvalRead = EvalRead
   { erPrecomputed :: Map Value Value
   , erLocalVars   :: Map Ident Value
+  , putFileFn     :: FilePath -> LBS.ByteString -> IO ()
```

Comment:
Minor nitpick: the other field selectors use the `er-` prefix, so I'd use them for these names too.

Reply:
I like it